### ISSUE TITLE
Images are loaded but the placeholder still showing 

### ISSUE THREAD

**Issue by yossefEl** [renefloor/flutter_cached_network_image#199]
thanks for this great package! this is a snippet of my code 
````
              CachedNetworkImage(
                          fit: BoxFit.cover,
                        height: 200,
                        width: MediaQuery.of(context).size.width,
                        color: Colors.white,
                        imageUrl:
                            "https://www.example.com/api/services_img/${data[index]['photo']}",
                        placeholder: (context, url) =>
                                    CircularProgressIndicator(),
                        errorWidget: (context, url, error) =>
                            new Icon(Icons.error),
                      ),
````

**Comment by itsJoKr** [comment]
Had the same problem. Reinstalled the app and now it works. 🤷‍♂

**Comment by thangvq95** [comment]
My issue: Works in debug. But not work in release.

**Comment by loumzy** [comment]
Had the same problem when I upgrade  flutter sdk to the lastest version v1.7.9

**Comment by loumzy** [comment]
@yossefEl  this is a problem with sqflutter， and has resolved in v1.1.6+1. You can redo 'Packages get' to resolved this problem

**Comment by yossefEl** [comment]
@loumzy @itsJoKr Thanks a  ot i just hot restarted the app and it works perfectly.
